pages/accountLists/[accountListId].page.tsx

```diff
@@ -111,12 +120,46 @@ export const getServerSideProps = makeGetServerSideProps(
       };
     } catch (error) {
       logErrorOnRollbar(error, '/accountLists/[accountListId].page');
-      return {
-        redirect: {
-          destination: '/',
-          permanent: false,
-        },
-      };
+
+      // Redirect to the account lists page if the error is anything except an
+      // account list not found error
+      const nonAccountListError =
+        error instanceof ApolloError &&
+        error.graphQLErrors.every(
+          (error) => !isAccountListNotFoundError(error),
```

Comment:
@canac Codecov wants me to test lines 128 and 129. Do you have any ideas on how to do that? 

It also wants coverage for many lines in client.ts. I'm not sure what to do there.